galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -167,7 +167,7 @@ public void runTest(Properties bootstrapProperties, Properties overridePropertie
                 //*** TODO remove above code in 0.9.0
             } catch (Exception e) {
                 logger.error("Unable to load stream " + stream + " settings", e);
-                updateStatus("finished", "finished");
+                updateStatus(TestRunLifecycleStatus.FINISHED, "finished");
```

Comment:
Why do we have FINISHED and "finished" both passed, as FINISHED.value() or similar gets the string "finished" ?
Seems there is something here which could be improved a bit.

Reply:
Gonna open another issue for this as it is a potentially larger refactoring process that deals with the dis store and how stuff is stored and read.